protos/IncomingPushMessageSignal.proto

```diff
@@ -143,19 +145,19 @@ message SyncMessage {
     optional uint64 timestamp = 2;
   }
 
-  message Settings {
+  message Configuration {
```

Comment:
Will probably require code changes.

Reply:
I didn’t find any references to `textsecure.protobuf.Settings` (or more generally, `Settings` that seem to be related to this). We do have `onSettings` and `messageReceiver.addEventListener('settings', onSettings);`. Should we change those to use `Configuration`?

Reply:
Hm. I do see this in `MessageReceiver`. The name of the field in `SyncMessage` was changed to reflect the new name of the type, so I think this will need to change:

```
        } else if (syncMessage.settings) {
            return this.handleSettings(envelope, syncMessage.settings);
        } else {
```

So yes, we might want to change to configuration to reduce confusion in the future.

Reply:
Thanks, just saw your comment. Fixed it: 91612880a5bf2c9ae8a9334877ac24e91102b905